localstack/services/apigateway/integration.py

```diff
@@ -73,6 +78,40 @@ def get_passthrough_behavior(passthrough_behaviour: str):
         return getattr(PassthroughBehavior, passthrough_behaviour, None)
 
 
+class IntegrationSubtypes(Enum):
+    def __new__(cls, value: str, implemented: bool):
+        obj = object.__new__(cls)
+        obj._value_ = value
+        obj.implemented = implemented
+        return obj
+
+    def invoke(self, api_context: ApiInvocationContext):
+        if not self.implemented:
```

Comment:
@whummer the API I was going for looks like this 
```
integration_subtype = IntegrationSubtypes(integr_details["IntegrationSubtype"])
return integration_subtype.invoke(invocation_context)
```
So the `implemented` is required because the `IntegrationSubtypes` is a match by the `value`, meaning is a valid integration subtype and is implemented. I agree if it was just a match value comparison (e.g "SQS-SendMessage" exists in the Dict or Enum), I would not have the enum with the value and will default. Not the case here, all the values are enumerated but only a couple are dispatching to the right backend integration.

As the static `dict`, or a `tuple` as it is now, not much of a difference - a method `is_implemented` that returns `bool` and selects the right dispatch integration that can be patch also seems a complete rewrite of the method, because you would be able to patch the state of the `dict` only, but the full method implementation. 

If misunderstood your suggestion, please correct me or dm